doc/tutorial/climada_engine_Impact.ipynb

```diff
@@ -341,12 +509,12 @@
        "</div>"
       ],
       "text/plain": [
-       "          value                    geometry  latitude  longitude  region_id  \\\n",
-       "0  4.730991e+10  POINT (-82.37500 23.12500)    23.125 -82.375000        192   \n",
-       "1  2.990876e+10  POINT (-82.29167 23.12500)    23.125 -82.291667        192   \n",
-       "2  6.839380e+09  POINT (-82.20833 23.12500)    23.125 -82.208333        192   \n",
-       "3  1.925246e+09  POINT (-82.12500 23.12500)    23.125 -82.125000        192   \n",
-       "4  2.422598e+08  POINT (-82.04167 23.12500)    23.125 -82.041667        192   \n",
+       "          value                    geometry   latitude  longitude  region_id  \\\n",
+       "0  1.077368e+05  POINT (-81.37500 21.70833)  21.708333 -81.375000        192   \n",
+       "1  1.671873e+06  POINT (-81.54167 21.62500)  21.625000 -81.541667        192   \n",
+       "2  3.421208e+06  POINT (-82.95833 21.87500)  21.875000 -82.958333        192   \n",
+       "3  1.546590e+07  POINT (-82.87500 21.87500)  21.875000 -82.875000        192   \n",
+       "4  7.168305e+07  POINT (-82.79167 21.87500)  21.875000 -82.791667        192   \n",
```

Comment:
What's happening here? Why do exposure values change by several orders of magnitude, along with the coordinates? I get the same result on my machine. Seems like these results were from early 2021. But what changed since then? All parameters in the function calls are the same.

Any ideas, @emanuel-schmid @tovogt?

Reply:
This is not related to this PR. It even happens when rolling back the source code to the last time that this notebook has been changed (in early 2022). I don't have time to dig any deeper, unfortunately. But when looking at the plot output below this cell, the changes are quite moderate (but at least big enough to spot differences with your bare eyes). I would suggest to continue this discussion somewhere else, on Slack or in a separate GitHub issue.

Reply:
Great, thanks for the quick feedback! 🙌